plugins/repository-s3/src/main/java/org/opensearch/repositories/s3/StatsMetricPublisher.java

```diff
@@ -20,18 +23,67 @@ public class StatsMetricPublisher {
 
     private final Stats stats = new Stats();
```

Comment:
we are not using `stats` anywhere . can we remove it ?

Reply:
Done